v5_MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,42 @@
+# Migrating to Unleash-Client-Dotnet 5.0.0
+
+This guide highlights the key changes you should be aware of when upgrading to v5.0.0 of the Unleash client.
+
+## Custom strategy changes
+
+Custom strategies no longer provide the option to access the constraints in their interface. The method `bool IsEnabled(Dictionary<string, string> parameters, UnleashContext context, IEnumerable<Constraint> constraints)` no longer exists. Other than that, custom strategies remain unchanged.
+
+## Direct access to feature toggles
+
+Direct access to the feature toggle objects through `UnleashClient.FeatureToggles` has been removed. All classes related to the internal representation of feature toggles is no longer publicly accessible in the SDK.
```

Comment:
> All classes related to the internal representation of feature toggles is no longer publicly accessible in the SDK.

```suggestion
Direct access to the feature toggle objects through `UnleashClient.FeatureToggles` has been removed. All classes related to the internal representation of feature toggles are no longer publicly accessible in the SDK.
```
?
Maybe I can't brain this properly

Reply:
No no, I think me no brain good. You brain good